dbuturu/Store-Manager
# #161236186 admin add product

#### What does this PR do?

Gives an admin the ability to add products.

#### Description of Task to be completed?

The endpoint 'POST /products' used to post products to the server.

#### How should this be manually tested?

Clone this repository 
    `git clone https://github.com/dbuturu/Store-Manager.git`
Check out this branch.
    `git checkout ch-Admin_add_product-API-#161236186`
Create a virtual environment 
    `virtualenv venv`
Active it by a run active command
     `source venv/bin/activate
      source venv/Scripts/activate
      venv\activate.bat
     `
Install dependencies
    `pip install -r requirements.txt`
Start the application.
    `python run.py`
Run Tests
    `pytest -v`

#### Any background context you want to provide?
This is an API chore
#### What are the relevant pivotal tracker stories?
#161236186
